src/constants.py

```diff
@@ -1,3 +1,6 @@
 IMAGE_ADDRESS = "https://raw.githubusercontent.com/cuappdev/assets/master/volume"
 STATES_LOCATION = "./.states/"
-PLACEHOLDER_IMAGE_ADDRESS = "https://raw.githubusercontent.com/cuappdev/assets/master/volume/placeholders/"
\ No newline at end of file
+PLACEHOLDER_IMAGE_ADDRESS = (
+    "https://raw.githubusercontent.com/cuappdev/assets/master/volume/placeholders/"
+)
+FILTERED_WORDS = ["covid-19", "coronavirus"]
```

Comment:
does this not include "mask" and "masks" ?

Reply:
I think since that "mask" and "masks" can be verbs, it would be too aggressive. Since this filter scans article bodies, it would be a pretty safe bet that there would be instances of the other filterable words in there.